<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agregarProductos</class>
 <widget class="QDialog" name="agregarProductos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Producto</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Orden:</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_estado">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="txt_orden">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>118</y>
     <width>1011</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="tab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Res</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Res</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_res">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>1011</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Pollo</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pollo</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_pollo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>1011</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Cerdo</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cerdo</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_cerdo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>1011</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Camarones</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Camarones</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_camarones">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>1011</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Pescado</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pescado</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_pescado">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>1011</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Platos Mixtos</string>
    </attribute>
    <widget class="QListWidget" name="list_mixtos">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>1011</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Platos Mixtos</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Platos Típicos</string>
    </attribute>
    <widget class="QListWidget" name="list_tipicos">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>1011</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Platos Típicos</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Extras</string>
    </attribute>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Extras</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_extras">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>1011</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>Bebidas</string>
    </attribute>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Bebidas</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_bebidas">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>1011</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>85</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fecha:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>35</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cliente:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butt_agregar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="txt_cliente">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>35</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butt_view_order">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="txt_fecha">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>85</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butt_cancel">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>480</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>list_res</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>butt_agregar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_pollo</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>butt_agregar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_cerdo</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>butt_agregar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_camarones</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>butt_agregar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_mixtos</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>butt_agregar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_tipicos</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>butt_agregar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_pescado</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>butt_agregar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_extras</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>butt_agregar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_bebidas</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>butt_agregar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
